lib/conjur/command/plugin.rb

```diff
@@ -0,0 +1,135 @@
+#
+# Copyright (C) 2014 Conjur Inc
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+require 'conjur/command'
+require 'yaml'
+
+GEMBIN = '/opt/conjur/embedded/bin/gem'
+
+class Conjur::Command::Plugin < Conjur::Command
+  desc 'Manage plugins'
+  command :plugin do |cmd|
+    # Only enable this plugin for omnibus-installed CLI
+    hide_docs(cmd) unless File.file?(GEMBIN) or ENV['DEV']
```

Comment:
It would be even better to test whether this file not only exists, but that's it's the actual current Ruby:

http://stackoverflow.com/a/17754480/953770

    ruby -e 'puts RbConfig.ruby`